backend/migration_tools/management/commands/drop_v2_schema.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
+    help = "Drop v2 schema if it exists"
+
+    def handle(self, *args, **kwargs):
+        schema_name = os.getenv("V2_SCHEMA_NAME", "unstract_v2")
+        with connection.cursor() as cursor:
+            cursor.execute(f"DROP SCHEMA IF EXISTS {schema_name} CASCADE")
+        self.stdout.write(
+            self.style.SUCCESS(f'Schema "{schema_name}" deleted successfully.')
+        )
```

Comment:
@muhammad-ali-e . Why are we dropping the V2 schema? What is the use case of this?

Reply:
This one I added to help to revert changes if we spot any issues during migrations. If we need to remove one schema before running the the migrations. But I think we should add a check like `if it not public schema`

Reply:
This is another command not couple with the migrate command